tests/unit/amo/components/TestSearchPage.js

```diff
@@ -48,44 +48,49 @@ describe('<SearchPage />', () => {
 
   it('renders a Paginate', () => {
     const root = render();
-    const paginator = findByTag(root, Paginate);
-    expect(paginator.props.count).toEqual(80);
-    expect(paginator.props.currentPage).toEqual(3);
-    expect(paginator.props.pathname).toEqual('/search/');
-    expect(paginator.props.queryParams).toEqual({ q: 'foo' });
+    const paginator = root.find(Paginate);
+    expect(paginator.prop('count')).toEqual(80);
```

Comment:
`toEqual` seems to still to strict checks for things like `true` and `80` though, right? Is there a reason not to use it for checks against simple types like this? Will `expect(80).toEqual('80');` pass? If not it'd be nice to always use the same API for equality checks. I always hated having to switch between `.equal()`, `.deepEqual()`, and `.strictEqual()` in the past...